tf2_ros/src/buffer.cpp

```diff
@@ -47,7 +47,7 @@
 namespace tf2_ros
 {
 
-Buffer::Buffer(rclcpp::Clock::SharedPtr clock, tf2::Duration cache_time, bool debug) :
+Buffer::Buffer(rclcpp::Clock::SharedPtr clock, tf2::Duration cache_time) :
```

Comment:
This is a bit of an API change but it's been legacy and unused for many rosdistros now. It's probably worth removing.